object_detection/utils/metrics.py

```diff
@@ -17,6 +17,7 @@
 from __future__ import division
 
 import numpy as np
+from six.moves import range
```

Comment:
Could you change this to `from six import moves` and use moves.range(...) below? This avoids redefining the built-in range function.